src/content/api/cli.mdx

```diff
@@ -190,6 +190,22 @@ npx webpack configtest ./webpack.config.js
 
 Path to your webpack configuration file. Defaults to `./webpack.config.js`.
 
+### Serve
+
+Run the webpack dev server.
+
+```bash
+npx webpack serve [options]
+```
+
+**example**
+
+```bash
+npx webpack serve --hot --live-reload
```

Comment:
I think better to use `--static` here, because it is more popular

Reply:
lets `--static` and `--open`: 

```suggestion
npx webpack serve --static --open
```